libs/keycloak-admin-client/src/utils/constants.ts

```diff
@@ -1,3 +1,3 @@
-export const defaultBaseUrl = "http://127.0.0.1:8180";
+export const defaultBaseUrl = "http://127.0.0.1:8080";
```

Comment:
This should not have been changed, please revert it.
```suggestion
export const defaultBaseUrl = "http://127.0.0.1:8180";
```

Reply:
Sorry for this.